lib/membrane_h264_plugin/parser.ex

```diff
@@ -269,41 +423,117 @@ defmodule Membrane.H264.Parser do
       }
 
       if Enum.any?(au, &(&1.status == :error)) or state.skip_until_keyframe do
+        Membrane.Logger.debug("skipping")
```

Comment:
I would write something more in that debug print

Reply:
it was my debugging log, shouldn't be here